pkg/storaged/anaconda.jsx

```diff
@@ -34,15 +34,31 @@
     return a.replace("-", "").toUpperCase() == b.replace("-", "").toUpperCase();
 }
 
-export function export_mount_point_mapping() {
+function device_name(block) {
+    // Prefer symlinks in /dev/stratis/.
+    return (block.Symlinks.map(decode_filename).find(n => n.indexOf("/dev/stratis/") == 0) ||
+            decode_filename(block.PreferredDevice));
+}
+
+export function remember_passphrase(block, passphrase) {
     if (!client.in_anaconda_mode())
         return;
 
-    function device_name(block) {
-        // Prefer symlinks in /dev/stratis/.
-        return (block.Symlinks.map(decode_filename).find(n => n.indexOf("/dev/stratis/") == 0) ||
-                decode_filename(block.PreferredDevice));
+    if (!window.isSecureContext)
+        return;
+
+    try {
+        const passphrases = JSON.parse(window.sessionStorage.getItem("cockpit_passphrases")) || { };
+        passphrases[device_name(block)] = passphrase;
+        window.sessionStorage.setItem("cockpit_passphrases", JSON.stringify(passphrases));
+    } catch {
+        console.warn("Can't record passphrases");
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19924-20240212-163533-e482442c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)